edx_proctoring/api.py

```diff
@@ -1695,7 +1695,7 @@ def _calculate_allowed_mins(exam, user_id):
     return allowed_time_limit_mins
 
 
-def _get_proctored_exam_context(exam, attempt, user_id, course_id, is_practice_exam=False):
+def _get_proctored_exam_context(exam, attempt, user_id, course_id, is_practice_exam=False, provider=None):
```

Comment:
Instead of passing in the provider, create the instance of it here and add the items from `_get_proctored_exam_view` and `_get_practice_exam_view`. Those functions only create the backend in order to add stuff to the context.